Celeste.Mod.mm/Patches/MapData.cs

```diff
@@ -115,6 +125,13 @@ public patch_MapData(AreaKey area)
             return orig_StartLevel() ?? Levels[0];
         }
 
+        public new LevelData Get(string levelName) {
```

Comment:
shouldn't this be tagged with MonoModReplace?

Reply:
done!